neuro_san/internals/run_context/langchain/llms/default_llm_factory.py

```diff
@@ -86,9 +82,6 @@ def __init__(self, config: Dict[str, Any] = None):
         self.llm_factories: List[LangChainLlmFactory] = [
             StandardLangChainLlmFactory()
         ]
-        self.llm_client_factories: List[LangChainLlmClientFactory] = [
-            StandardLangChainLlmClientFactory()
-        ]
```

Comment:
Abandon the separate factory for clients creation (Nouns), now that we are doing these policy clients (Verbs).